service/worker/processor.go

```diff
@@ -443,18 +443,10 @@ func (p *replicationTaskProcessor) updateFailureMetric(scope int, err error) {
 
 func (p *replicationTaskProcessor) isTransientRetryableError(err error) bool {
 	switch err.(type) {
-	case *h.ShardOwnershipLostError:
-		return true
-	case *shared.ServiceBusyError:
-		return true
-	case *shared.LimitExceededError:
-		return true
-	case *shared.InternalServiceError:
-		return true
-	case *shared.RetryTaskError:
-		return true
-	default:
+	case *shared.BadRequestError:
```

Comment:
process also deserializes the payload.  Any deserialization error should also cause the message to immediately land in DLQ.